review.md

```diff
@@ -2,48 +2,72 @@
+
+1. Is a [public member](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-your-membership-in-organizations/publicizing-or-hiding-organization-membership) of one of the trusted [GitHub organizations](https://docs.github.com/en/organizations/collaborating-with-groups-in-organizations/about-organizations) listed at <https://github.com/r-multiverse/contributions/blob/main/organizations>.
+
+and if the [pull request](https://github.com/r-multiverse/contributions/pulls) itself:
+
+1. Was [created by the GitHub web interface](contributors.md).
+1. Adds new contributed listings to the [`packages` folder](https://github.com/r-multiverse/contributions/tree/main/packages).
+1. Does not add, modify, or delete any other files in <https://github.com/r-multiverse/contributions>.
+
+and if each new [contributed listing](https://github.com/r-multiverse/contributions/tree/main/packages):
+
+1. Is a single line of text with a valid HTTPS URL.
+1. Points to an existing public GitHub/GitLab repository.
+
+and if the contributed GitHub/GitLab repository:
+
+1. Includes a GitHub/GitLab [release](https://docs.github.com/en/repositories/releasing-projects-on-github/about-releases).
+1. Includes an R package at the top level whose package name is the same as the repository name.
+1. Is listed in the `URL` field of the corresponding CRAN page (if a package with the same name is on CRAN).
+
+and if the R package:
+
+1. Includes a license from the "Recommended licenses" section at the end of this policy.
+1. Does not have an advisory in the [R Consortium Advisory Database](https://github.com/RConsortium/r-advisory-database).
+1. Is not part of the CRAN mirror at <https://github.com/cran>.
+
+## Manual review
+
+R-multiverse [moderators](governance.md#moderator) review [pull requests](https://github.com/r-multiverse/contributions/pulls) that the bot flags for manual review.
+The [moderator](governance.md#moderator) inspects the package for compliance with R-multiverse policies, including but not limited to [Acceptable Use](aup.md), [Terms of Use](terms.md), and [Code of Conduct](conduct.md).
+The moderator accepts the contribution if and only if it complies with all policies.
+
+## Removal
+
+R-multiverse may remove a package from its own repositories at any time if the package violates R-multiverse policies.
+
+## Modification
+
+No R-multiverse staff member (administrator, moderator, or otherwise) may modify a package in R-multiverse without the explicit consent of the owners declared in the license of the package.
+
+## Recommended licenses
+
+The [Acceptable Use Policy](aup.md) prohibits packages that "violate any applicable laws, regulations, or third-party rights, including intellectual property rights".
+In practice, this means each package must have a valid open-source license.
```

Comment:
```suggestion
If a package has a valid open-source license, this ensures that distribution by R-multiverse does not violate the intellectual property rights of the package authors.
```

Reply:
On second thought, I think we need to  require a valid OSS license. Otherwise, we don’t know if R-multiverse has permission to distribute the package. We already have one package with “Licence: TBD”. We will need clear policy language that prohibits this if we are going to enforce proper guardrails.

Reply:
@shikokuchuo, I am proposing we codify this via https://github.com/wlandau/r-multiverse.github.io/commit/275a247fa0487c42d27cd3bd96e5e12b471c8f6a. When we say "protect the intellectual property rights of the package authors", we also mean "protect R-multiverse staff from the appearance of copyright infringement".